bindings/python/Jamfile

```diff
@@ -33,41 +33,11 @@ if $(LIBTORRENT_PYTHON_INTERPRETER)
 	using python : : "$(LIBTORRENT_PYTHON_INTERPRETER)" : : : <libtorrent-python>on ;
 }
 
-# copied from boost 1.63's boost python jamfile
-rule find-py3-version
-{
-	local BOOST_VERSION_TAG = [ modules.peek boostcpp : BOOST_VERSION_TAG ] ;
-	if $(BOOST_VERSION_TAG) >= 1_67
-	{
-		# starting with boost 1.67.0 boost python no longer define a separate
-		# target for python3 (boost_python3) so then we just use the regular
-		# boost_python target
-		return ;
-	}
-	local versions = [ feature.values python ] ;
-	local py3ver ;
-	for local v in $(versions)
-	{
-		if $(v) >= 3.0
-		{
-			py3ver = $(v) ;
-		}
-	}
-	return $(py3ver) ;
-}
-
 if $(BOOST_ROOT)
 {
 	use-project /boost : $(BOOST_ROOT) ;
 	alias boost_python : /boost/python//boost_python : : : <include>$(BOOST_ROOT) ;
-	if [ find-py3-version ]
-	{
-		alias boost_python3 : /boost/python//boost_python3 : : : <include>$(BOOST_ROOT) ;
-	}
-	else
-	{
-		alias boost_python3 : boost_python ;
-	}
+	alias boost_python3 : boost_python ;
```

Comment:
I think the `boost_python3` target can be removed altogether with this change. I'll try out an update to this PR